l1-contracts/src/core/Rollup.sol

```diff
@@ -57,6 +57,8 @@ struct SubmitEpochRootProofInterimValues {
   uint256 endBlockNumber;
   Epoch epochToProve;
   Epoch startEpoch;
+  bool isFeeCanonical;
```

Comment:
Moving this into the interim struct since I could not run coverage due to the stack :)